flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/stream/table/TableEnvironmentITCase.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.stream.table
+
+import java.io.File
+
+import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
+import org.apache.flink.table.api.scala.StreamTableEnvironment
+import org.junit.Test
+import org.apache.flink.api.scala._
+import org.apache.flink.table.api.internal.TableEnvironmentImpl
+import org.apache.flink.table.api.{EnvironmentSettings, TableEnvironment}
+import org.apache.flink.table.api.scala._
+import org.apache.flink.table.descriptors.{FileSystem, OldCsv, Schema}
+import org.apache.flink.table.expressions.utils.UDFWithJobParameterChecking
+import org.apache.flink.table.planner.StreamPlanner
+import org.apache.flink.table.runtime.utils.CommonTestData
+
+class TableEnvironmentITCase {
```

Comment:
Use the available test utils for ITCases. ITCases are very expensive. I'm also wondering if this class is really necessary or if this test can be integrated into an existing class.

Reply:
> Use the available test utils for ITCases. ITCases are very expensive. 

In the latest commit it has inherited AbstractTestBase. 

> I'm also wondering if this class is really necessary or if this test can be integrated into an existing class.

IMO to verifying changes in this PR, actual job execution is necessary because we should make sure the UDFs running in flink runtime can read configurations which set in `TableConfig`. It does not depend on any operator. I noticed that for `BatchTableEnvironment` the test cases of this type are placed in `org.apache.flink.table.runtime.batch.table.TableEnvironmentITCase`. So I think for `StreamTableEnvironment` it might be a reasonable choice to put the same test cases in `org.apache.flink.table.runtime.stream.table.TableEnvironmentITCase`. 
Does this make sense to you?